geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloDataStore.scala

```diff
@@ -652,16 +652,14 @@ class AccumuloDataStore(val connector: Connector,
    * @param featureName
    * @return
    */
-  private def getAttributes(featureName: String) =
-    metadata.read(featureName, ATTRIBUTES_KEY)
+  private def getAttributes(featureName: String) = metadata.read(featureName, ATTRIBUTES_KEY)
 
   /**
    * Reads the feature encoding from the metadata. Defaults to TEXT if there is no metadata.
    */
   def getFeatureEncoding(sft: SimpleFeatureType): FeatureEncoding = {
-    val encodingString = metadata.read(sft.getTypeName, FEATURE_ENCODING_KEY)
-                         .getOrElse(FeatureEncoding.TEXT.toString)
-    FeatureEncoding.withName(encodingString)
+    metadata.read(sft.getTypeName, FEATURE_ENCODING_KEY)
+      .map(FeatureEncoding.withName(_)).getOrElse(FeatureEncoding.TEXT)
```

Comment:
[style] In this sort of situation, you can omit the explicit argument, and use the shorthand, ```.map(FeatureEncoding.withName)```.